pkg/ChaosErr/common_errors.go

```diff
@@ -0,0 +1,48 @@
+// Copyright 2022 Chaos Mesh Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package ChaosErr
```

Comment:
>please consider rename package name to chaoserr

ok

>I found that these errors only be used in pkg/chasodaemon/tasks, consider moving them into this package

These errors can be used in chaosdaemon or other place.I put it here because there not have a global error package in  chaosdaemon.

Reply:
Fixed.